evadb/third_party/databases/mysql/mysql_handler.py

```diff
@@ -109,3 +110,23 @@ def execute_native_query(self, query_string: str) -> DBHandlerResponse:
             return DBHandlerResponse(data=self._fetch_results_as_df(cursor))
         except mysql.connector.Error as e:
             return DBHandlerResponse(data=None, error=str(e))
+
+    def _mysql_to_python_types(self, mysql_type: str):
+        mapping = {
+            "char": str,
+            "varchar": str,
+            "text": str,
+            "boolean": bool,
+            "integer": int,
+            "int": int,
+            "float": float,
+            "double": float,
+            # Add more mappings as needed
+        }
+
+        if mysql_type in mapping:
+            return mapping[mysql_type]
+        else:
+            raise Exception(
+                f"Unsupported column {mysql_type} encountered in the mysql table. Please raise a feature request!"
```

Comment:
Can you also add the link to our Slack in this message (https://evadb.ai/community)?

Reply:
Add `self.connection = None` in the base class. Else it throws property not found in some cases.

Reply:
What do you mean by the link?